int32 id
geometry_msgs/Vector3 robot_position
geometry_msgs/Vector3[] points_msg
geometry_msgs/Vector3 ransac_normal
geometry_msgs/Vector3 axis
float64 radius
